src/plonk/circuit.rs

```diff
@@ -27,6 +28,11 @@ impl<C: ColumnType> Column<C> {
     pub(crate) fn column_type(&self) -> &C {
         &self.column_type
     }
+
+    pub(crate) fn hash(&self, hasher: &mut Blake2bState) -> [u8; 64] {
+        hasher.update(&format!("{:?}", self).as_bytes());
+        hasher.finalize().as_bytes().try_into().unwrap()
+    }
```

Comment:
This function shouldn't return a hash, it should just update the hasher it is given.